apollo-integration/src/test/graphql/com/apollographql/android/impl/normalizer/HeroNameResponse.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It is identical, just thought for organizational purposes 1 graphql query = 1 graphql response of the same name.